aqt/metadata.py

```diff
@@ -41,6 +41,21 @@
 class SimpleSpec(SemanticSimpleSpec):
     pass
 
+    @staticmethod
+    def usage() -> str:
+        return (
+            "See documentation at: "
+            "https://python-semanticversion.readthedocs.io/en/latest/reference.html#semantic_version.SimpleSpec\n"
+            "Examples:\n"
+            '* "*": matches everything\n'
+            '* "5": matches every version with major=5\n'
+            '* "5.6": matches every version beginning with 5.6\n'
+            '* "5.*.3": matches versions with major=5 and patch=3\n'
+            '* "<5": matches every version with major<5\n'
+            '* ">5,<6.2": matches versions starting with 6.0 and 6.1\n'
+            '* "<=5": matches major<=5, including all preview/prerelease versions'
+        )
```

Comment:
We haven't discussed this at all, but `semantic_version` also provides NpmSpec as an alternative to SimpleSpec. Their docs at https://python-semanticversion.readthedocs.io/en/latest/reference.html#semantic_version.SimpleSpec mention an extension for PyPI-style specs as well, but I haven't been able to locate it yet.

The reason I mention this is that if we use a spec standard that everyone else already knows about, we won't need this `usage()` method to tell them how to use it. I could be very wrong about this, but I think that more of our users will know about npm specs than will know how to use SimpleSpec. I would rather not try to teach people how to use SimpleSpec if there's an alternative that they already know.

Reply:
IMO, we can drop description following example.

```
            '* "<5": matches every version with major<5\n'
            '* ">5,<6.2": matches versions starting with 6.0 and 6.1\n'
            '* "<=5": matches major<=5, including all preview/prerelease versions'
```

These are mostly a description of simple spec.

Reply:
It is no harm but IMHO many users may not use these complex expressions.

Reply:
Ok, I removed the last 3; if users need more help they can read the docs.